app/src/main/java/com/nervousfish/nervousfish/activities/MainActivitySorter.java

```diff
@@ -134,24 +122,17 @@ private void hideNoContactsContent() {
     private void sortOnKeyType() {
         final ExpandableListView ev = (ExpandableListView) this.mainActivity.findViewById(R.id.expandable_contact_list_by_key_type);
```

Comment:
Could you change the name of this local variable to something more descriptive 😇